### PR TITLE
new command

### PR DESCRIPTION
**Pull request by deodev**

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -40,6 +40,12 @@ Discord.on('message', function (message) {
   	}
 });
 
+Discord.on('message', function (message) {
+	if (message.content === '@everyone'
```
Comment (tonkku107) [inline bot.js]: You're missing something :eyes:

**Comment by deodev** [comment]
resolved now

File: bot.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ Discord.on('message', function (message) {
 });
 
 Discord.on('message', function (message) {
-	if (message.content === '@everyone'
+	if (message.content === '@everyone')
```
Comment (tonkku107) [inline bot.js]: Still missing something

File: bot.js [hunk 3]
```diff
@@ -41,7 +41,7 @@ Discord.on('message', function (message) {
 });
 
 Discord.on('message', function (message) {
-	if (message.content === '@everyone'
+	if (message.content === '@everyone')
 	    return message.reply('@everyone ping ping @everyone')
 	}
```
Comment (tonkku107) [inline bot.js]: Or this isn't supposed to be here

**Pull request merged**